packages/app/src/messages/presenters/message.presenter.ts

```diff
@@ -63,15 +63,14 @@ export class MessagePresenter {
   private static getMessageType(
```

Comment:
And also, please made the same changes in `packages/app/src/data-sync/presenters/message/message.presenter.ts`

Reply:
Added in [005e09b](https://github.com/mudita/mudita-center/pull/981/commits/005e09bffe30858f82fce48686253fc8915cc4ca), thanks!